accord-core/src/test/java/accord/impl/basic/Journal.java

```diff
@@ -88,20 +82,6 @@ public void shutdown()
         this.node = null;
     }
 
-    @Override
-    public <R> void handle(LocalRequest<R> message, BiConsumer<? super R, Throwable> callback, Node node)
```

Comment:
Why is the local request path able to go away now along with the associated propagate messages to record in the journal?

Reply:
The journal records outcomes now, so we don't need to save messages.